doc/content/reference/adr/_index.md

```diff
@@ -140,6 +140,130 @@ The end device can also be steered towards the 125kHz channels, which usually ar
 ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-settings.adr.mode.dynamic --mac-settings.adr.mode.dynamic.channel-steering.lora-narrow
 ```
 
+### Configure NbTrans for an End Device per data rate
+
+{{< new-in-version "3.30.0">}}
+
+{{% tts %}} allows you fine-grained access to limit NbTrans per data rate. When setting this value, make sure that the minimum NbTrans is not above the maximum NbTrans or vice versa.
```

Comment:
The backend errors if one tries this. I don't think it needs to be explicitly stated.